### PR TITLE
typo fix

### PR DESCRIPTION
**Pull request by NicDominguez**
[ASTRO-1644](https://rocketcom.atlassian.net/browse/ASTRO-1644)

Fixes typo in compliance guideline 3.4.4.

### CODE REVIEW

File: _content/_data/compliance.json [hunk 1]
```diff
@@ -1,949 +1,949 @@
...
-              "figure": "/img/patterns/status-system-fundamentals.png",
-              "status": "new"
-            },
-            {
-              "numeral": 2,
-              "rule": "Elements indicating state or status shall use only the specified [Status Colors](/patterns/status-system/#status-colors).",
-              "status": "new"
-            }
-          ]
+    "version": "3.1.0",
```
Comment (amalyah) [inline _content/_data/compliance.json:2]: Does the version number go up with edits or does that come later in the process?

File: _content/_data/compliance.json [hunk 2]
```diff
@@ -1,949 +1,949 @@
...
+    "contents": [
+        {
+            "name": "General",
+            "numeral": 1,
+            "rules": [
+                {
+                    "numeral": 1,
+                    "rule": "ASTRO UXDS provides developers with guidance to develop user interfaces. User interfaces must compy with the guidance in this section to be EGS Design Compliant. Developers are encouraged to conform to guidance in other sections identified as “Rules of Thumb”, “Do's” and “Don'ts” as described on individual component pages.",
```
Comment (amalyah) [inline _content/_data/compliance.json:10]: Nice to have: compy > comply

File: _content/_data/compliance.json [hunk 3]
```diff
@@ -1,949 +1,949 @@
...
+                        },
+                        {
+                            "numeral": 2,
+                            "rule": "Help Text shall be left-aligned under the relevant form element, regardless of location of the form element’s label. The only exception is for Radio Button and Checkbox list item Help Text which shall be left-aligned to the item’s text and not its icon.",
+                            "status": "new"
+                        },
+                        {
+                            "numeral": 3,
 
-              "rule": "Required fields shall be marked with an asterisk to the right of the label when the majority of a form is optional.",
```
Comment (amalyah) [inline _content/_data/compliance.json:134]: There are a lot of removals going on in this PR. Should I assume that this is just prettier tidying things up but the content is still all there?

**Review by amalyah — APPROVED** [review]

File: _content/_data/compliance.json (hunk 3 above)
Comment (NicDominguez) [reply _content/_data/compliance.json:134]: Yes, prettier decided to shift everything. Sorry about that, I know it's hard to review when that happens.

File: _content/_data/compliance.json (hunk 1 above)
Comment (NicDominguez) [reply _content/_data/compliance.json:2]: I believe that will come when we publish a release, this is still the draft branch.

**Review by amalyah — APPROVED** [review]

**Review by nortonprojects — APPROVED** [review]

**Pull request merged**